data/menus.yaml

```
@@ -38,6 +38,14 @@
     - Undergraduate maths: undergrad.html
     - Wiedijk's 100 theorems: 100.html
 
+- title: Theories
+  items:
+    - Category Theory: theories/category_theory.html
+    - Linear Algebra: theories/linear_algebra.html
+    - Natural Numbers: theories/naturals.html
+    - Sets and finite sets: theories/sets.html
+    - Topology: theories/topology.html
```

Not sure what the best ordering is, but certainly I would put natural numbers at the top and categories at the bottom.
```suggestion
    - Natural Numbers: theories/naturals.html
    - Linear Algebra: theories/linear_algebra.html
    - Sets and finite sets: theories/sets.html
    - Topology: theories/topology.html
    - Category Theory: theories/category_theory.html
```